ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -207,6 +219,13 @@ def get_postgroups(post, cdump='gdas'):
         fhout_hf = post['FHOUT_HF_GFS']
         fhrs_hf = list(range(fhmin, fhmax_hf + fhout_hf, fhout_hf))
         fhrs = fhrs_hf + list(range(fhrs_hf[-1] + fhout, fhmax + fhout, fhout))
+    elif cdump in ['gefs']:
```

Comment:
This looks to be duplicate of 'gfs'. Maybe just add 'gefs' to that list unless/until we create a separate set of variables?